src/js/drop.js

```diff
@@ -124,6 +124,23 @@ function createContext(options={}) {
         throw new Error('Drop Error: You must provide a target.');
       }
 
+      var dataPrefix = 'data-' + drop.classPrefix;
+
+      var contentAttr = this.target.getAttribute(dataPrefix);
```

Comment:
Seems like we can DRY this up a little and put these in a loop. That way, we can add support for additional options easily.